bot.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Code for the telegram-bot track."""
```

Comment:
generally we don't use such stuff in our Python modules since they're pretty self-explanatory. Are you sure this is helpful?